### PR TITLE
Screen portfolio buyer

### PR DESCRIPTION
**Pull request by ngvtuan**
##### Commits
- [x] Screen 5.5 - Portfolio Buyer

##### FreedCamp
- [x] [Screen 5.5 - Portfolio Buyer](https://freedcamp.com/view/2956183/tasks/40217557)

### CODE REVIEW

**Review by ihor-stack — COMMENTED** [review]
Need to solve conflict issue

File: src/common/components/NFTCard/NFTCard.tsx [hunk 1]
```diff
@@ -2,12 +2,19 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 import cn from 'classnames'
 
+import styles from './NFTCard.module.css'
+
 export interface INFTCompactCardProps {
```
Comment (muhadovic) [inline src/common/components/NFTCard/NFTCard.tsx:7]: Please use the Type instead of the Interface.

**Review by muhadovic — CHANGES_REQUESTED** [review]

**Review by muhadovic — CHANGES_REQUESTED** [review]

**Review by CrazyOptimist — APPROVED** [review]
Looks good to me.

**Comment by CrazyOptimist** [comment]
Already merged by [PR#144](https://github.com/pastelnetwork/pastel-electron-wallet/pull/144)

**Pull request closed without merging**